include/bluetooth/att.h

```diff
@@ -43,6 +43,9 @@ extern "C" {
 #define BT_ATT_ERR_PROCEDURE_IN_PROGRESS	0xfe
 #define BT_ATT_ERR_OUT_OF_RANGE			0xff
 
+/* Version 5.2, Vol 3, Part F, 3.2.9 defines maximum attribute size to 512 */
+#define BT_ATT_MAX_ATTRIB_SIZE			512
```

Comment:
Should be  `BT_ATT_MAX_ATTRIBUTE_SIZE`, it's not that much longer, and we use the full spelling other places (see above).

Is LEN better to use than SIZE?

Reply:
Changed to `ATTRIBUTE` rather than `ATTRIB`. The spec defines the value as `length`, so also changed `SIZE` to `LEN`.